css/style.css

```diff
@@ -455,5 +457,14 @@ footer {
 
   .footer-filter {
     grid-column-start: 1;
+    padding-top: 20px;
   }
+
+  .footer-toggle-buttons {
+    grid-template-columns: 1fr;
+  }
+}
+
+.unimplemented {
+  display: none;
```

Comment:
This seems a good idea for some cases at least. I guess there might be exceptions.